### PR TITLE
Add 2024Q3 UVC report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: website/content/en/status/report-2024-07-2024-09/uvc.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This work is to create FreeBSD UVC (USB Video Class) kernel driver and follow
+v4l2 APIs, so that most of the Linux camera applications can be easily ported
+to FreeBSD.
+
+The code is still cleaning up and will be submitted to official review after completing.
+
+Current Status:
+
+1. The key functions of UVC driver is enabled.
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/uvc.adoc:18]: s/ is / are /

s/ of / of the /

**Review by pauamma — APPROVED** [review]
Approved with these nits fixed

**Review by pauamma — APPROVED** [review]
LGTM

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**